index.d.ts

```diff
@@ -0,0 +1,72 @@
+import { EventEmitter } from 'events';
+
+export interface Options{
+    statsInterval?: number;
+}
+
+export interface Bot{
+    botid: string;
+    votes: number;
+    servers: string;
+    monthly_votes: number;
+    owners: string[];
+    longdesc: string;
+    shortdesc: string;
+    certified: string;
+    links: {
+        invite: string;
+        supportserver?: string;
+        website?: string;
+        github?: string;
+        donate?: string;
+    }
+}
+
+export interface BotReviews{
+    reviews: {
+        rating: string;
+        responses: any[];
+        flagged: boolean;
+        createdAt: string;
+        userid: string;
+        botid: string;
+        message: string;
+    }[];
+}
+
+export interface BotAnalytics{
+    build: string;
+    analytics: {
+        date: string;
+        unique_views: number;
+        views: number;
+        votes: number;
+        reviews: number;
+        referrers: {
+            name: string;
+            value: number;
+        }[];
+        countries: {
+            name: string;
+            value: number;
+        }[];
+    }[];
+}
+
+export type Response<T> = T | { message: string } | { error: string };
+
+export class VoidBots extends EventEmitter{
+
+    public token: string;
+
+    public constructor(token: string, options: any, client: any);
```

Comment:
```suggestion
    public constructor(token: string, options?: any, client?: any);
```
`options` and `client` parameters are optional.